java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/model/Result.java

```diff
@@ -0,0 +1,13 @@
+package com.google.cloud.model;
+
+public class Result {
```

Comment:
Would you add javadoc? Very likely this class should have a better name. I don't see the result of dependency analysis in this class.

Reply:
Done.